proposals/revised-governance.md

```diff
@@ -0,0 +1,227 @@
+# A Proposal for the Governance Structure of the Node.js Project
+
+## What problems are being addressed?
+
+* The TSC as it exists today is largely disfunctional.
+ 
+## tl;dr version
+
+This proposal eliminates the TSC as a standing body of individual voting
+members and instead distributes the decision making authority previously
+held by that standing body to the chartered working groups.
+
+This proposal does not change the CTC in any way other than in terms of how
+new working groups are chartered.
+
+## Proposal
+
+What I am proposing is a revised TSC structure such that:
+
+1. Eliminate the TSC standing membership body. Rather than individual voting
+   TSC members, each chartered working group has a single voting seat. It is
+   up to each working group to decide on its own process for exercising that
+   vote.
+2. An Admin team would be created that would be charged with handling the
+   more mundane administrative tasks currently handled by the TSC (budget
+   allocations, github org management, handling confidential information).
+   Tasks like administering the travel fund could be handed off to the
+   Community Committee. This admin team would not be a voting body.
```

Comment:
Generally, yes. The members of the Admin team would have github org ownership rights.